go.mod

```diff
@@ -2,6 +2,8 @@ module github.com/spiffe/spire
 
 go 1.20
 
+replace github.com/spiffe/spire-api-sdk => github.com/guilhermocc/spire-api-sdk v0.0.0-20230405150557-559bee4eb02e
```

Comment:
Remove this replacement after https://github.com/spiffe/spire-api-sdk/pull/39 is merged

Reply:
This has been merged :)